_rules/html-page-lang-valid-bf051a.md

```diff
@@ -19,83 +19,41 @@ authors:
 
 ## Applicability
 
-The root element of the [web page](https://www.w3.org/TR/WCAG21/#dfn-web-page-s), if it is an `html` element with a `lang` and/or `xml:lang` attribute that is neither empty ("") nor only [whitespace](#whitespace).
+This rule applies to any [document element](https://dom.spec.whatwg.org/#document-element) if it is an `html` element that:
```

Comment:
As before, this has gotten quite technically dense, making it difficult to read. Saying things like that the content type is a property of node document adds unnecessary complexity. The content-type definition already describes explains this. Similarly with top-level browser context. That can only mean one thing, we don't need to specify that it's a type of browser context.

Reply:
See https://github.com/act-rules/act-rules.github.io/pull/934#discussion_r329934298.